test/unit/vdom/elements.js

```diff
@@ -0,0 +1,34 @@
+import element from '../../lib/element';
+import { symbols, vdom } from '../../../src/index';
+
+const shadowDomV0 = !!document.createElement('div').createShadowRoot;
```

Comment:
In this case it would just use Chrome's native implementation. Underneath the hood it would still write a `<content>` element when the user outputs a slot using `vdom('slot')` or `vdom.slot()`. I see your point, though. In terms of performance, trying to use a native implementation first will be better than using a polyfill. Maybe it's worth inverting the check to use a v1 flag, but then it's likely Chrome would use a polyfill when it has native support, even if it's v0. Thoughts?